aarch64_extension.cpp

```diff
@@ -142,6 +185,111 @@ class AArch64ArchitectureExtension : public ArchitectureHook {
     return 0;
   }
 
+  ExprId liftMemOperand(LowLevelILFunction& il, cs_arm64_op& op) {
+    uint32_t Rn = this->m_base->GetRegisterByName(
+        cs_reg_name(disassembler.Get(), op.mem.base));
+    int32_t imm = op.mem.disp;
+
+    size_t Rn_size = this->m_base->GetRegisterInfo(Rn).size;
+
+    assert(8 == Rn_size); /* Otherwise capstone was seriously wrong */
```

Comment:
I want to avoid assertions as much as possible, so we should probably have a `std::optional<ExprId>` and then return an empty set here on error.